qutebrowser/browser/commands.py

```diff
@@ -992,7 +992,7 @@ def tab_move(self, index: typing.Union[str, int]=None, count=None):
             self._tabbed_browser.setUpdatesEnabled(True)
 
     @cmdutils.register(instance='command-dispatcher', scope='window',
-                       maxsplit=0)
+                       maxsplit=0, no_replace_variables=True)
```

Comment:
Hmm, while this makes sense (as the command needs to replace them *after* splitting the commandline) this adds the respective line to the documentation, which is really confusing... Not sure what's the best way around this though?